src/HawksearchVue.js

```diff
@@ -584,6 +598,36 @@ class HawksearchVue {
         }
     }
 
+    static getClientData(store) {
+        var clientData;
```

Comment:
The clientData should not be empty if it is always attached to the parameters object. In this case, it is is probably best to define it with only the visitor id and if available - the language.